lib/components/molecules/Toggle/Toggle.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+// @flow
+/**
+ *
+ * Toggle
+ *
+ */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { InputVanilla as Input } from '../../atoms/Input';
+import Label from '../../atoms/Label';
+
+import styles from './Toggle.style';
+
+const Toggle = ({
+  id,
+  name,
+  type,
+  disabled,
+  className,
+  children,
+  labelProps,
+  labelOn,
+  labelOff,
+  labelName,
+  ...otherProps
+}) => {
+  const [toggleOn, handleToggle] = useState(false);
+
+  let toggleLabel = '';
+  if (toggleOn) {
+    toggleLabel = labelOn;
+  } else toggleLabel = labelOff;
+
+  return (
+    <div className={className}>
+      <span className="labelName">{labelName}</span>
+      <section>
+        <Label htmlFor={id} {...labelProps}>
+          <Input
```

Comment:
I dont think it is following the right structure. 
Can the toggle component be part of a Form. It may, right ?
We should create a FieldToggle as well. How that will be structured can be discussed with Vinod or me